models/models_file.py

```diff
@@ -0,0 +1,168 @@
+            config = {}
+        return cls.fromDict(config)
+
+
+class Method(Enum):
+    GET = "GET"
+    POST = "POST"
+    PUT = "PUT"
+    DELETE = "DELETE"
+    PATCH = "PATCH"
+
+
+class Response:
+    def __init__(self, status_code: str, body: str, method: str, latency: int) -> None:
+        logger = logging.getLogger("main")
+        self.body = body
+        self.latency = latency
+        try:
+            self.status_code = HTTPStatus(int(status_code))
+        except ValueError as e:
+            logger.error("Your config file passed invalid status code: %s", e)
+            raise SystemExit
+        try:
+            self.method = Method(method.upper())
+        except ValueError as e:
+            logger.error("Your config file passed invalid method: %s", e)
+            raise SystemExit
+
+    def __repr__(self) -> str:
+        return f"Response {self.method} after {self.latency} {self.status_code}: {self.body}"
+
+    @classmethod
+    def fromDict(cls, data: dict, method: str) -> object:
+        body = data["body"]
+        statusCode = data["statusCode"]
+        method = method
+        latency = data["latency"]
+        return cls(statusCode, body, method, latency)
+
+
+class Endpoint:
+    def __init__(self, endpoint: str, responses: "List[Response]") -> None:
+        self.responses = responses
+        self.endpoint = "/" + endpoint
+
+    def __repr__(self) -> str:
+        return f"Endpoint: \"{self.endpoint}\" responses: {self.responses}"
+
+    @classmethod
+    def fromDict(cls, data: dict) -> object:
+        endpoint = cls.adapt_route(data['endpoint'])
+        response = data["responses"][0]
+        responses = [Response.fromDict(response, data["method"])]
+        return cls(endpoint, responses)
+
+    @classmethod
+    def responseFromDict(cls, data: dict) -> object:
+        response = data["responses"][0]
+        return Response.fromDict(response, data["method"])
+
+    @classmethod
+    def adapt_route(cls, route: str) -> str:
+        name_allowed = "A-z0-9_"
+        params_search = re.compile(f"(:[{name_allowed}]+)")
+        params = params_search.findall(route)
+        for param in params:
+            route = route.replace(param, f"<{param[1:]}>", 1)
+        return route
+
+    def view_maker(self) -> object:
+        def view_func(*args, **kwargs):
+            for response in self.responses:
+                if request.method == response.method.value:
+                    sleep(response.latency / 1000)
+                    body = response.body
+                    param_dict = {"urlParam": kwargs, "queryParam": request.args}
+                    params = ReSearching.search_params(body)
+                    for param in params:
+                        total = param[0]
+                        type = param[1]
+                        var_name = param[2].strip()[1:-1]
+                        default = param[3][1:-1]
```

Comment:
to możesz rozpakować już w forze 
`for total, type, var_name, default in params:`